src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/IHtmlHelper.cs

```diff
@@ -133,7 +135,7 @@ public interface IHtmlHelper
             string controllerName,
             object routeValues,
             FormMethod method,
-            bool suppressAntiforgery,
+            bool? suppressAntiforgery,
```

Comment:
Why would such an overload be useful? Why is it useful to make `suppressAntiforgery` nullable? (Hint, it should _suppress_ antiforgery when `true` and leave the system to its own whims when `false`.)